package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "cssnano": "^4.0.0",
     "del-cli": "^1.0.0",
-    "jest": "^22.0.0",
+    "jest": "^23.6.0",
```

Comment:
Could you drop this change here and instead commit it in #175?

Reply:
@michael-ciniawsky sure!